README.rst

```diff
@@ -76,11 +76,23 @@ Contributing
 To contribute to pika, please make sure that any new features or changes
 to existing functionality **include test coverage**.
 
-*Pull requests that add or change code without coverage will most likely be rejected.*
+*Pull requests that add or change code without coverage will be rejected.*
```

Comment:
@lukebakken, we need to be policing test coverage more effectively. For example, the Twisted connection adapter was accepted without test coverage in 2011 and still has no meaningful test coverage in 2018. Over the years it has been anywhere between partially broken and completely broken. My preference in general is not to release code without meaningful test coverage.